app/workers/users/bust_cache_for_space_change_worker.rb

```diff
@@ -0,0 +1,14 @@
+module Users
+  # Conditionally delete the async cache when a Space changes
+  #
+  # @note Good news for those of you using Redis as your cache store, you have access to `Rails.cache.delete_matched`
+  #
+  # @see https://api.rubyonrails.org/classes/ActiveSupport/Cache/RedisCacheStore.html#method-i-delete_matched
+  class BustCacheForSpaceChangeWorker < BustCacheBaseWorker
+    def perform
+      return unless Rails.cache.respond_to?(:delete_matched)
+
+      Rails.cache.delete_matched("^#{AsyncInfoController::ASYNC_INFO_CACHE_KEY_PREFIX}-")
```

Comment:
This string must follow [Redis glob patterns](https://redis.io/commands/keys/), which are not compatible with any common regex patterns. They're only slightly more powerful than MS-DOS file-glob patterns. :joy:

```suggestion
      Rails.cache.delete_matched("#{AsyncInfoController::ASYNC_INFO_CACHE_KEY_PREFIX}-*")
```